visual-tests/tests/change-rows-order.spec.js

```diff
@@ -0,0 +1,45 @@
+import { helpers } from '../imports/helpers';
+
+const { test, expect } = require('@playwright/test');
+
+/* ======= */
+
+const stylesToAdd = [
+  helpers.cssFiles.cookieInfo,
+  helpers.cssFiles.dynamicDataFreeze
+];
+
+test(helpers.testTitle(__filename), async({ page }, workerInfo) => {
+  helpers.init(workerInfo, process);
```

Comment:
```suggestion
  helpers.init(workerInfo);
```
[#ref](https://github.com/handsontable/handsontable/pull/10165/files#r1068189649)

Reply:
probably it's kind of leftover. Modified.